doc/sphinx/changes.rst

```diff
@@ -6,6 +6,470 @@ Recent changes
 
    .. include:: ../unreleased.rst
 
+Version 8.11
+------------
+
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+The main changes brought by |Coq| version 8.11 are:
+
+- `Ltac2`__, a new tactic language for writing more robust larger scale
+  tactics, with built-in support for datatypes and the multi-goal tactic monad.
+- `Primitive floats`__ are integrated in terms and follow the binary64 format
+  of the IEEE 754 standard, as specified in the `Coq.Float.Floats` library.
+- `Cleanups`__ of the section mechanism, delayed proofs and further
+  restrictions of template polymorphism to fix soundness issues related to
+  universes.
+- New `unsafe flags`__ to disable locally guard, positivity and universe
+  checking. Reliance on these flags is always printed by
+  :g:`Print Assumptions`.
+- `Fixed bugs`__ of :g:`Export` and :g:`Import` that can have an impact
```

Comment:
I suggest making this warning even more visible:

```suggestion
- `Fixed bugs`__ of :g:`Export` and :g:`Import` that can have a **significant impact**
```

We could also add a `.. warning::` box to the detailed entry below.